### DISCUSSION TITLE
open API (swagger v3) for Django REST Framework ??

### DISCUSSION THREAD

**Discussion by ghost** [encode/django-rest-framework#8456] [Ideas & Suggestions]
https://swagger.io/tools/swagger-ui/

Swagger UI is used in python FastAPI framework. I found it to be very useful.
How about implementing this in the future version Django REST ?

**Comment by bespokebob** [comment]
OpenAPI is already implemented. See https://www.django-rest-framework.org/api-guide/schemas/#generating-an-openapi-schema

**Comment by jairhenrique** [comment]
Consider using the https://github.com/tfranzel/drf-spectacular.
